package.json

```diff
@@ -42,6 +42,7 @@
   "peerDependencies": {
     "@commitlint/cli": ">=8",
     "@commitlint/config-conventional": ">=8",
-    "@commitlint/prompt-cli": ">=8"
+    "@commitlint/prompt-cli": ">=8",
+    "@weahead/commitlint-plugin-weahead": ">=0.x"
```

Comment:
Do we need `.x` in the version? Isn't `>=0` enough?

Reply:
True!
Wasn't sure how we wanted to lock that dependency, updating the commit. 👍